<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1149</width>
    <height>834</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: #3B3B3B;</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>1111</width>
      <height>691</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <property name="leftMargin">
      <number>0</number>
     </property>
     <property name="topMargin">
      <number>0</number>
     </property>
     <item row="2" column="1">
      <widget class="QScrollArea" name="timeline_area">
       <property name="styleSheet">
        <string notr="true">margin: 0 20px 37px;
padding: 5px;
border-radius: 20px;
background-color: #2D2D2D;</string>
       </property>
       <property name="widgetResizable">
        <bool>true</bool>
       </property>
       <widget class="QWidget" name="scrollAreaWidgetContents_2">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>347</width>
          <height>481</height>
         </rect>
        </property>
        <widget class="QTableWidget" name="timeline_table">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>381</width>
           <height>451</height>
          </rect>
         </property>
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: #2D2D2D;
border-radius: 30px;
color: white; 
</string>
         </property>
        </widget>
       </widget>
      </widget>
     </item>
     <item row="0" column="0">
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="0">
        <widget class="QLabel" name="title_label">
         <property name="font">
          <font>
           <pointsize>40</pointsize>
           <weight>75</weight>
           <bold>true</bold>
           <kerning>true</kerning>
          </font>
         </property>
         <property name="text">
          <string>Structurly</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLabel" name="subtitle_label">
         <property name="text">
          <string>Organization on a new level</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignBottom|Qt::AlignLeading|Qt::AlignLeft</set>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="create_new_Task_label">
       <property name="font">
        <font>
         <pointsize>18</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">margin-top: 20px;
padding:10px;
color: #EA526F;</string>
       </property>
       <property name="text">
        <string>Neuen Task anlegen</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLabel" name="label">
       <property name="font">
        <font>
         <pointsize>18</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">margin-top: 20px;
color: #EA526F;</string>
       </property>
       <property name="text">
        <string>TextLabel</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="2" column="2">
      <layout class="QVBoxLayout" name="verticalLayout_right">
       <item alignment="Qt::AlignTop">
        <widget class="QCalendarWidget" name="calendarWidget_2">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="autoFillBackground">
          <bool>false</bool>
         </property>
         <property name="gridVisible">
          <bool>false</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="2" column="0">
      <layout class="QVBoxLayout" name="create_task_group">
       <property name="leftMargin">
        <number>20</number>
       </property>
       <property name="topMargin">
        <number>20</number>
       </property>
       <property name="rightMargin">
        <number>20</number>
       </property>
       <item>
        <widget class="QLineEdit" name="task_name_edit">
         <property name="styleSheet">
          <string notr="true">border: none;
color: #eee;</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="placeholderText">
          <string>Task Name</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTextEdit" name="description_edit">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="styleSheet">
          <string notr="true">color: #eee;</string>
         </property>
         <property name="placeholderText">
          <string>Beschreibung</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="start_time_label">
         <property name="styleSheet">
          <string notr="true">color: #eee;</string>
         </property>
         <property name="text">
          <string>Startzeit</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTimeEdit" name="start_time_edit">
         <property name="styleSheet">
          <string notr="true">color: #eee;
border: none;
margin: 0 20px;
</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="time_needed_label">
         <property name="styleSheet">
          <string notr="true">color: #eee;</string>
         </property>
         <property name="text">
          <string>Dauer</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSlider" name="time_needed_edit">
         <property name="cursor">
          <cursorShape>OpenHandCursor</cursorShape>
         </property>
         <property name="styleSheet">
          <string notr="true"/>
         </property>
         <property name="maximum">
          <number>40</number>
         </property>
         <property name="value">
          <number>2</number>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="category_label">
         <property name="styleSheet">
          <string notr="true">color: #eee;</string>
         </property>
         <property name="text">
          <string>Kategorie</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="category_dropdown">
         <property name="styleSheet">
          <string notr="true">margin: 0 20px;</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="priority_label">
         <property name="styleSheet">
          <string notr="true">color: #eee;</string>
         </property>
         <property name="text">
          <string>Priorit√§t</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="priority_edit">
         <property name="styleSheet">
          <string notr="true">color: #eee;
border: none;
margin-bottom: 10px;</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="create_task_btn">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="styleSheet">
          <string notr="true">color: black;
height: 40px;
background-color: rgba(192, 139, 43, 255);
border-radius: 10px;

</string>
         </property>
         <property name="text">
          <string>Task anlegen</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="printTasks">
         <property name="text">
          <string>print Tasks</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="status_label">
         <property name="styleSheet">
          <string notr="true">color: lightgreen;
margin-top: 15px;

</string>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1149</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuStructurly">
    <property name="title">
     <string>Structurly</string>
    </property>
   </widget>
   <addaction name="menuStructurly"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
